src/entities/SelectedMovies/model/slices/selectedMoviesSlices.test.ts

```diff
@@ -0,0 +1,43 @@
+import { SelectedMovieSchema } from 'entities/SelectedMovies';
+import {
+  addMovie,
+  clearStateMovies,
+  deleteMovie,
+  selectedMovieSlice,
+} from './selectedMoviesSlices';
+import { Movie } from 'shared/types';
+
+const testMovie: Movie = {
+  Poster: 'https://www.kinopoisk.ru/picture/2836590/',
+  Title: 'Super Men',
+  Type: 'movie',
+  Year: 1978,
+  imdbID: '123456',
+};
+
+test('testing clear reducer', () => {
+  const previousState: SelectedMovieSchema = {
+    movies: [],
+  };
+  expect(selectedMovieSlice.reducer(previousState, clearStateMovies())).toEqual(
+    { movies: [] }
+  );
```

Comment:
```suggestion
  expect(selectedMovieReducer(previousState, clearStateMovies())).toEqual(
    { movies: [] }
  );
```
вот тут можно заменить, ты же экспортируешь редусер из слайса
в большинстве случаем даже нет необходимости слайс экспортировать из файла, обычно достаточно его редусера и экшена